apps/s_client.c

```diff
@@ -456,6 +456,7 @@ typedef enum OPTION_choice {
     OPT_USE_SRTP, OPT_KEYMATEXPORT, OPT_KEYMATEXPORTLEN, OPT_PROTOHOST,
     OPT_MAXFRAGLEN, OPT_MAX_SEND_FRAG, OPT_SPLIT_SEND_FRAG, OPT_MAX_PIPELINES,
     OPT_READ_BUF, OPT_KEYLOG_FILE, OPT_EARLY_DATA, OPT_REQCAFILE,
+    OPT_TFO,
```

Comment:
#ifdef here?

Reply:
Not really necessary... the actual option definition (`-tfo`) is removed if TCP_FASTOPEN is not defined, so it can't be used if the option is not available.
If this is `#ifdef`'d out, then any reference also has to be `#ifdef`d out, making the code harder to read/follow, IMO.